src/services/users.service.ts

```diff
@@ -36,13 +20,20 @@ export class UsersService {
     return this.userRepository.findOne(id);
   }
 
+  async findUserByEmail(email: string) {
+    return await User.createQueryBuilder('user')
```

Comment:
same as the case above. Please remove the return await statement. The explanation is in an earlier comment

Reply:
fixed